string query
---
bool found
string label
geometry_msgs/Pose pose
float32 confidence